### DISCUSSION TITLE
default_prop for ui.chip doesn't work

### DISCUSSION THREAD

**Discussion by v479038280** [zauberzeug/nicegui#3114] [General]
```py
ui.chip.default_props('color="positive" text-color="white"')
ui.chip().props('label="REMOTE"').props('color="positive"')
ui.chip().props('label="LOCAL"')
```
the color doesn't work ，I‘m not sure if this is considered a problem.
here is the rusult
![微信截图_20240523001829](https://github.com/zauberzeug/nicegui/assets/25096045/ce3923a0-1a04-4d0b-84e8-ab9f12f2dee8)

**Comment by falkoschindler** [comment]
Hi @v479038280,

The color of a `ui.chip` is controlled by its `color` parameter, which defaults to "primary". So even if you set default props, when initializing a `ui.chip` the color is overwritten.
